packages/node/test/onunhandledrejection.test.ts

```diff
@@ -1,15 +1,29 @@
-import { Hub, init, Integrations, Scope } from '../src';
+import { Scope } from '@sentry/core';
+import { Hub } from '@sentry/hub';
```

Comment:
For my learning -- Does changing this import affect the test, or is it just cosmetic/preference?

Reply:
Cosmetic preference